docker/image.ts

```diff
@@ -67,22 +67,14 @@ export interface ImageRegistry {
  */
 export class Image extends pulumi.ComponentResource {
     /**
-     * The base image name that was built and pushed.  This does not include the digest annotation, so is not pinned to
-     * the specific build performed by this docker.Image.
+     * The base image name that was built and pushed.  This does not include the id annotation, so
+     * is not pinned to the specific build performed by this docker.Image.
      */
     public baseImageName: pulumi.Output<string>;
     /**
-     * The pinned image name, including digest annotation.
+     * The unique pinned image name on the remote repository.
      */
     public imageName: pulumi.Output<string>;
-    /**
-     * The built image Id.
-     */
-    public id: pulumi.Output<string>;
-    /**
-     * The pushed image digest.
-     */
-    public digest: pulumi.Output<string | undefined>;
```

Comment:
These are breaking changes.  Can we mitigate these by deprecating these properties and publishing something sufficiently correct for these two values?

Reply:
yup.   i'll do that.